cgi/search.pl

```diff
@@ -665,11 +665,11 @@
 		}
 	}
 
-	if (defined $sort_by) {
-		$current_link .= "&sort_by=$sort_by";
-	}
+	# if (defined $sort_by) {
+	# 	$current_link .= "&sort_by=$sort_by";
+	# }
 
-	$current_link .= "\&page_size=$limit";
+	# $current_link .= "\&page_size=$limit";
```

Comment:
I'm not sure we want to remove it because we may let user sort on whatever they want through the API.

Reply:
In search.pl we also have different sort possibilities.

So I would say: use add_param here also, and maybe verify first that sort_by or page_size is not already in current_link.